content/actions/using-workflows/workflow-syntax-for-github-actions.md

```diff
@@ -413,7 +413,7 @@ Selects an action to run as part of a step in your job. An action is a reusable
 
 We strongly recommend that you include the version of the action you are using by specifying a Git ref, SHA, or Docker tag number. If you don't specify a version, it could break your workflows or cause unexpected behavior when the action owner publishes an update.
 - Using the commit SHA of a released action version is the safest for stability and security.
-- Using the specific major action version allows you to receive critical fixes and security patches while still maintaining compatibility. It also assures that your workflow should still work.
+- If the action publishes major version tags, using them allows you to receive critical fixes and security patches while still maintaining compatibility.
```

Comment:
I'd lean towards:
```suggestion
- If the action publishes major version tags, using them should allow you to receive critical fixes and security patches while still retaining compatibility (within the abilities of the publisher).
```

In that it's almost worth noting that "yes, an action author may try to not break things, but, they're probably human, so they might anyway".

Reply:
I agree with you completely, and my first draft was more explicit about that fact, but I thought a more minimal change may be more likely to be accepted. Happy to incorporate this change or something similar, but let's see whether GitHub finds bandwidth to review this pull request.

Reply:
👋 I'll add this suggestion to the PR, and then check with an engineering team to confirm the preferred wording